rts/sra/src/main/java/io/narayana/sra/demo/api/TripController.java

```diff
@@ -112,7 +112,7 @@ void finiController() {
     @Produces(MediaType.APPLICATION_JSON)
     // end = false because we want the SRA to be associated with a booking until the user confirms the booking
     @SRA(value = SRA.Type.REQUIRED, end = false)
-    public Response bookTrip( @HeaderParam(RTS_HTTP_CONTEXT_HEADER) String sraId,
+    public Response bookTrip( @HeaderParam(RTS_HTTP_CONTEXT_HEADER) @DefaultValue("") String sraId,
```

Comment:
We cant set a default value for sraId as per Tom, because a transaction can't have a "default" ID. It needs to be allocated by the transaction manager.